test/lit.cfg

```diff
@@ -19,6 +19,7 @@
 # -----------------------------------------------------------------------------
 
 from __future__ import absolute_import
+import json
```

Comment:
I've forgotten whether we can assume this everywhere. I sure hope so.

Reply:
I think we can ignore this, thanks for spotting it was from early iterations of the module fixing that folks were doing for the Dispatch overlay

Reply:
I think we don't need it after all, removing -- this was from the ongoing experiments by others to fix the modules which is now fixed.

Removing the import.